packages/react-focus/package.json

```diff
@@ -51,6 +51,7 @@
     "@uifabric/set-version": "^7.0.6",
     "@uifabric/styling": "^7.10.17",
     "@uifabric/utilities": "^7.13.2",
+    "keyboard-key": "1.0.1",
```

Comment:
You should update everyone to 1.1.0 which has types. Trouble is that the types only work properly if you have `esModuleInterop` enabled, which fluent doesn't (Levi or Shift could give details of why). The current typing format is necessary to properly reflect the code's assignment to `module.exports`...though come to think of it, you could just update keyboard-key to a more interop-friendly export format. It's here: https://github.com/levithomason/keyboard-key/blob/master/src/keyboardKey.js

Reply:
I agree, we should update everyone. @layershifter and @levithomason why doesn't Fluent have `esModuleInterop`?

Reply:
There's more background here (`allowSyntheticDefaultImports` and `esModuleInterop` are closely related): https://github.com/microsoft/fluent-ui-react/pull/1395